.gitignore

```diff
@@ -0,0 +1 @@
+**/COLCON_IGNORE
```

Comment:
I'm afraid it won't be this PR pioneering on the .gitignore file ;) please keep this to your local workspace